vnext/Microsoft.ReactNative/Fabric/Composition/UiaHelpers.cpp

```diff
@@ -214,4 +214,24 @@ ExpandCollapseState GetExpandCollapseState(const bool &expanded) noexcept {
   }
 }
 
+void AddSelectionItemsToContainer(CompositionDynamicAutomationProvider *provider) noexcept {
+  IRawElementProviderSimple *selectionContainer;
+  provider->get_SelectionContainer(&selectionContainer);
+  if (!selectionContainer)
+    return;
+  auto selectionContainerProvider = static_cast<CompositionDynamicAutomationProvider *>(selectionContainer);
+  auto simpleProvider = static_cast<IRawElementProviderSimple *>(provider);
+  selectionContainerProvider->AddToSelectionItems(simpleProvider);
+}
+
+void RemoveSelectionItemsFromContainer(CompositionDynamicAutomationProvider *provider) noexcept {
+  IRawElementProviderSimple *selectionContainer;
+  provider->get_SelectionContainer(&selectionContainer);
```

Comment:
Like Daniel says above, you'll probably want to put selectionContainer in a com_ptr here so that the ref will be released.

Reply:
For inspiration, see this bit of code above:

```
  winrt::com_ptr<IRawElementProviderFragmentRoot> spFragmentRoot;
  spFragment->get_FragmentRoot(spFragmentRoot.put());
```

Reply:
The parameter type of get_SelectionContainer has to be IRawElementProviderSimple**. The method is set by the UIA ISelectionProvider interface and we just have control over implementing it. 

I could set selectionContainer to be a com_ptr, but I believe I'd then have to cast the pointer to be IRawElementProviderSimple** to be passed as a parameter into get_SelectionContainer. And then would that wreck the memory releasing behavior anyway?

Reply:
Hi, so the code can look like this:
```
  winrt::com_ptr<IRawElementProviderSimple> selectionContainer;
  provider->get_SelectionContainer(selectionContainer.put());
```

That call to "put()" returns an object that acts like a IRawElementProviderSimple** and updates the selectionContainer object to have the new value.

And then, when the selectionContainer object goes out of scope (even if due to an exception being thrown), the com_ptr<> dtor will automatically call Release().  Voila!